### ISSUE TITLE
train_model.py weird output

### ISSUE THREAD

**Issue by Takachsin** [Sentdex/pygta5#15]
I had to recreate my issue because I didn't do a good job of explaining it before. I'm having an issue with train_model.py on Windows. I have linked the output below that I receive on epoch 8 (it's the same for the previous ones). It does not name the model files correctly (it saves them as model_alexnet-22.data-00000-of-00001.tempstate18218624911869073751). When I run the same script on the same set of code from my Macbook it works and correctly names my output file. My desktop is much faster than my Macbook though and I'd like to figure out why the data isn't saving correctly on Windows, so if you or anyone can provide any help, it'd be greatly appreciated. 

Oh and at the very beginning I get Scipy not supported! even though it's installed.

Training Step: 78  | total loss: [1m[32m0.58550[0m[0m | time: 2.035s
| Momentum | epoch: 008 | loss: 0.58550 - acc: 0.8028 -- iter: 064/664
[A[ATraining Step: 79  | total loss: [1m[32m0.55399[0m[0m | time: 4.048s
| Momentum | epoch: 008 | loss: 0.55399 - acc: 0.8151 -- iter: 128/664
[A[ATraining Step: 80  | total loss: [1m[32m0.52058[0m[0m | time: 6.050s
| Momentum | epoch: 008 | loss: 0.52058 - acc: 0.8276 -- iter: 192/664
[A[ATraining Step: 81  | total loss: [1m[32m0.49628[0m[0m | time: 8.038s
| Momentum | epoch: 008 | loss: 0.49628 - acc: 0.8356 -- iter: 256/664
[A[ATraining Step: 82  | total loss: [1m[32m0.48760[0m[0m | time: 10.098s
| Momentum | epoch: 008 | loss: 0.48760 - acc: 0.8380 -- iter: 320/664
[A[ATraining Step: 83  | total loss: [1m[32m0.45747[0m[0m | time: 11.487s
| Momentum | epoch: 008 | loss: 0.45747 - acc: 0.8495 -- iter: 384/664
[A[ATraining Step: 84  | total loss: [1m[32m0.45454[0m[0m | time: 12.883s
| Momentum | epoch: 008 | loss: 0.45454 - acc: 0.8520 -- iter: 448/664
[A[ATraining Step: 85  | total loss: [1m[32m0.45365[0m[0m | time: 14.871s
| Momentum | epoch: 008 | loss: 0.45365 - acc: 0.8502 -- iter: 512/664
[A[ATraining Step: 86  | total loss: [1m[32m0.44107[0m[0m | time: 16.870s
| Momentum | epoch: 008 | loss: 0.44107 - acc: 0.8495 -- iter: 576/664
[A[ATraining Step: 87  | total loss: [1m[32m0.41673[0m[0m | time: 18.863s
| Momentum | epoch: 008 | loss: 0.41673 - acc: 0.8583 -- iter: 640/664
[A[ATraining Step: 88  | total loss: [1m[32m0.60144[0m[0m | time: 22.472s
| Momentum | epoch: 008 | loss: 0.60144 - acc: 0.8069 | val_loss: 0.13101 - val_acc: 0.9660 -- iter: 664/664

**Comment by daveyman123** [comment]
http://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy
you have to install numpy+mkl for the scipy error
